src/test/java/org/example/JDBCExampleTest.java

```diff
@@ -0,0 +1,63 @@
+package org.example;
+
+import org.junit.AfterClass;
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class JDBCExampleTest {
+
+    @AfterClass
+    public static void closeDbConnection() {
+        JDBCExample.closeConnection();
```

Comment:
don't run tests on live production databases :) we will talk about this in class